packages/xstate-test/src/index.ts

```diff
@@ -102,6 +103,28 @@ export class TestModel<TTestContext, TContext> {
     return testPlans.filter(predicate);
   }
 
+  public getPlanFromEvents(
+    events: Array<EventObject>,
+    { target }: { target: StateValue }
+  ) {
+    const path = getPathFromEvents<TContext>(this.machine, events);
+
+    if (!path.state.matches(target)) {
+      throw new Error(
+        `The last state ${JSON.stringify(
+          (path.state as any).value
+        )} does not match the target: ${JSON.stringify(target)}`
+      );
+    }
+
+    return this.getTestPlans({
```

Comment:
Should I return an array here?

Reply:
after sleeping on it, decided to go with the single plan

Reply:
Yep, it's made obvious by the name